bindata/manifests/switchdev-config/files/bind-switchdev.sh.yaml

```diff
@@ -0,0 +1,35 @@
+mode: 0755
+overwrite: true
+path: "/usr/local/bin/bind-switchdev.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -eux
+    input="/etc/sriov_config.json"
+
+    if [ ! -f $input ]; then
+      echo "File /etc/sriov_config.json not exist."
+      exit
+    fi
+
+    # Required for NetworkManager configuration(e.g. bond) to settle down
+    sleep 3
```

Comment:
Can we use `nmcli` or `ip` command to check the link state, instead of using sleep?

Reply:
This may not be easy since we don't know what configuration the nmcli or ip should wait. there could be cases that additional bond or other NM configuration is not configured by user.